src/node.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Entry point for setting up of Requests Debugger Tool.
+ * Initiates actions such as processing of args, setting up loggers,
+ * initiating all connectivity checks and stats collection before starting
+ * the proxy tool.
+ */
+
+var constants = require('../config/constants');
+var LogFiles = constants.LOGS;
+var RdGlobalConfig = constants.RdGlobalConfig;
+var CommandLineManager = require('./commandLine');
+var ConnectivityChecker = require('./connectivity');
+var RdHandler = require('./server');
+var StatsFactory = require('./stats/statsFactory');
+var LogManager = require('./logger');
+var fs = require('fs');
+var path = require('path');
+var Utils = require('./utils');
+
+var RdTool = {
+
+  /**
+   * Initializes the Logging directory, Loggers & Stats Handlers.
+   */
+  initLoggers: function () {
```

Comment:
`initLoggers` method seems to be doing more than it should, deleting log files(if required) and init log dir. can we move/break it to a different init methods?

Reply:
Had kept it in one since it initializes logging directory as well for the loggers. Because they both are coupled. I can't have loggers without the logging directory... or .. I can't have logging directory if no loggers exists.
I can move it to an internal method? which will be called by the `initLoggers`... um.. will that help?

Reply:
>I can't have loggers without the logging directory

Sure. For that we can always order the initLogDir method called prior to initLogger.

> I needn't have logging directory if no loggers exists

are we checking this in current implementation?

Reply:
> node.js can be renamed. `app.js`/ `index.js` / `requestDebugger.js`

renamed to `requestsDebugger.js`

Reply:
2nd part considers that the logging directory will exist, thus, keeping them in the same function.

Reply:
approach i suggest is, call a single init method. call different inits from in there with required checks (dir exists or not). this way we can keep the SRP in place.

Reply:
done

Reply:
resolving.